src/menelaus/datasets/make_example_data.py

```diff
@@ -86,3 +88,29 @@ def make_example_batch_data():
     df["drift"] = df["year"].isin([2009, 2012, 2015, 2018, 2021])
     df.drop("index", axis=1, inplace=True)
     return df
+
+
+def find_git_root(search_dirs=(".git",)):
+    """Find the root directory for the git repo, so that we don't have to
+    fool with strange filepaths.
+
+    This is a strange place to have this utility function, but we don't have any
```

Comment:
I agree that it's a bit much to do all this for one function, but I still think it may be better to put this into a separate location, or at least a separate file. 

I prefer flatter structures wherever possible, so it may be nice to do e.g. (I couldn't say what the names should be)
- datasets/utils.py or
- datasets/_locate.py etc. etc. or
- utils/helper.py